docs/building-on-etherlink/endpoint-support.md

```diff
@@ -397,9 +397,14 @@ For information about these endpoints, see the Geth documentation at https://get
   </thead>
   <tbody>
     <tr>
-      <td>`debug_traceTransaction`</td>
+      <td>`debug_traceCall`</td>
       <td>Partially</td>
-      <td>Supported on testnet but not Mainnet</td>
+      <td>Supports only callTracer and structLogger, not custom call tracers</td>
```

Comment:
We could use the opportunity to mention after which block each is available on each network.